code/extensions/ScheduledExecutionExtension.php

```diff
@@ -18,16 +20,21 @@ class ScheduledExecutionExtension extends DataExtension {
 		'ExecuteFree'			=> 'Varchar',
```

Comment:
Hello @assertchris ,

thanks for your quick response. I left them as they are because I personally find it better readable. But that might be just me. Is there a convention somewhere? I couldn't find anything in the code style guide.

spek